docs/src/content/quickstart.md

```diff
@@ -4,4 +4,35 @@ name: Quickstart
 
 # Quickstart Guide
 
-This will lead you to different paths.
+This Quickstart tutorial will guide you to setup the Hydrus server, we will be using docker in this tutorial,
+Therefore you can go through the [Docker container engine](https://docker-curriculum.com/) and its best practices deploying virtualized environments.
+
+## Running hydrus server
+> Make sure you have docker-compose installed in your system
+
+#### Getting the [hydrus repository](https://github.com/HTTP-APIs/hydrus)
+```bash
+git clone https://github.com/HTTP-APIs/hydrus
+```
+#### Change directory and run docker-compose
```

Comment:
Having the steps as headings give the impression that there might be some sub-steps as well. I think having an ordered list for the steps can be a good thing. Did you have any special reason to list them as headings?